src/app.ts

```diff
@@ -67,6 +58,11 @@ export default (
     labelNames: ['install', 'operation', 'status', 'method'],
   });
 
+  const taskToMethod = new Map([
+    ['peribolos-run', 'schedulePushTask'],
+    ['peribolos-dump-config', 'scheduleDumpConfig]'],
```

Comment:
Thinking about this, this mapping is pointless, I think we should use the task name directly as a metric label.. WDYT?

Reply:
Yea I think that's a better solution than having the map.

Reply:
I don't think we need that. It's not a label, it's a value of a label `method`. I think it's actually better to keep the pairing 1 to 1 and don't do any magic there. Use the plain values please.

Reply:
Ok :+1: